src/brushes/eraser_brush.class.js

```diff
@@ -0,0 +1,592 @@
+(function () {
```

Comment:
Let's rename this file as eraser_mixin, and let's move it in the mixin folder, since it contains a brush and changes to canvas and object.

Reply:
DONE
c80f5645